scancodeio/static/add-inputs.js

```diff
@@ -21,7 +21,42 @@
 // Visit https://github.com/nexB/scancode.io for support and download.
 
 const fileInput = document.querySelector('#id_input_files');
-fileInput.onchange = () => {
+fileInput.onchange = update_files
+
+const input_urls = document.querySelector('#id_input_urls');
+input_urls.oninput = () => {
+  input_urls.style.height = "";
+  input_urls.style.height = input_urls.scrollHeight + 3 + "px";
+}
+
+// Handling DropZone Events
+const dropzone = document.querySelector('#file_upload_label');
+dropzone.addEventListener("dragenter", dragenter, false);
```

Comment:
What's the purpose of the `false` argument here?

Reply:
Nothing in particular. It's just the default syntax that I use. I'll remove this in the next commit.